tests/admin_changelist/tests.py

```diff
@@ -551,6 +551,53 @@ def test_show_all(self):
         cl.get_results(request)
         self.assertEqual(len(cl.result_list), 10)
 
+    def test_set_model_admin_field_help_text(self):
+        """
+        Ensure that a help_text property can be set to a custom field
```

Comment:
Just state the expected behavior rather than 'Ensure that...' (all tests ensure and test things).